app/src/forms/submission/controller.js

```diff
@@ -148,12 +149,12 @@ module.exports = {
           },
         },
         options: {
-          convertTo: 'pdf',
+          convertTo: convertTo,
           overwrite: true,
           reportName: fileName,
         },
         template: {
-          content: btoa(template.template),
+          content: template.template.toString(),
```

Comment:
One oops of a couple - we're storing the file in the DB base64 encoded. We shouldn't be storing it in the DB anyway, but it's one more thing to keep in mind when we move to the object storage.